test/unit.js

```diff
@@ -26,7 +26,62 @@ describe('bore', () => {
     expect(<Fn />.localName).to.equal('div');
   });
 
-  it('setting attributes', () => {
+  it('setting attributes on CustomElement', () => {
```

Comment:
right, actually that is in next test. totally missed that ( also because non obvious naming of attrs `test1` and so on :P )